core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/project/ProjectServiceClient.java

```diff
@@ -33,8 +33,8 @@
     /**
      * Get all projects in current workspace.
      *
-     * @param workspaceId
-     *         id of current workspace
+     * @param devMachine
+     *         of current devMachine
```

Comment:
mb just something like `instance of current dev machine`? Or maybe we can use not current dev machine here